packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.tsx

```diff
@@ -57,18 +69,43 @@ export class RefreshPicker extends PureComponent<Props> {
   }
 
   render() {
-    const { onRefresh, intervals, tooltip, value, text, isLoading, noIntervalPicker, width } = this.props;
+    const {
+      onRefresh,
+      intervals,
+      tooltip,
+      value,
+      text,
+      isLoading,
+      noIntervalPicker,
+      width,
+      offOptionLabelMsg,
+      offOptionAriaLabelMsg,
+      offDescriptionAriaLabelMsg,
+      onDescriptionAriaLabelMsg,
```

Comment:
**Nit:** Maybe I would add a comment above these props

Currently I feel that it's not so easy to understand what the exact purpose of these props are without reading through the code, and it can also be confusing that we only have a single prop for the "on" scenario and three for the "off" one.